crash-reporter/init.sls

```diff
@@ -0,0 +1,55 @@
+{% from 'common/map.jinja' import common %}
+{% from tpldir ~ '/map.jinja' import tracker %}
+
+include:
+  - common
+  - python
+
+crash-reporter:
+  virtualenv.managed:
+    - name: /home/servo/crash-reporter/_venv
+    - venv_bin: virtualenv-3.5
+    - python: python3
+    - system_site_packages: False
+    - require:
+      - pkg: python3
+      - pip: virtualenv
+  pip.installed:
+    - pkgs:
+      - git+https://github.com/servo/crash-reporter@{{ tracker.rev }}
+    - bin_env: /home/servo/crash-reporter/_venv
+    - upgrade: True
+    - require:
+      - virtualenv: crash-reporter
+  {% if grains.get('virtual_subtype', '') != 'Docker' %}
```

Comment:
This conditional (and the corresponding endif) can be removed like in https://github.com/servo/saltfs/pull/1006.